runelite-client/src/main/java/net/runelite/client/plugins/mta/enchantment/EnchantmentRoom.java

```diff
@@ -34,22 +36,23 @@
 import net.runelite.api.Player;
 import net.runelite.api.Tile;
 import net.runelite.api.TileItem;
+import net.runelite.api.Perspective;
 import net.runelite.api.coords.WorldPoint;
 import net.runelite.api.events.GameStateChanged;
-import net.runelite.api.events.GameTick;
 import net.runelite.api.events.ItemDespawned;
 import net.runelite.api.events.ItemSpawned;
 import net.runelite.client.eventbus.Subscribe;
 import net.runelite.client.plugins.mta.MTAConfig;
 import net.runelite.client.plugins.mta.MTARoom;
+import net.runelite.client.ui.overlay.OverlayUtil;
 
 @Slf4j
 public class EnchantmentRoom extends MTARoom
 {
 	private static final int MTA_ENCHANT_REGION = 13462;
 
 	private final Client client;
-	private final List<WorldPoint> dragonstones = new ArrayList<>();
+	private final List<Tile> dragonstones = new ArrayList<>();

```

Comment:
Why are tiles used here instead of WorldPoints like before? We only ever call `getLocalLocation()` of dragonstones.

Reply:
I originally did this when testing, but forgot to revert it before pushing the changed. Reverted it back to WorldPoints in the latest commit.